### DISCUSSION TITLE
Sunbird ED Provision on AWS Infra

### DISCUSSION THREAD

**Discussion by muzafferjoya** [Sunbird-Ed/Community#719] [Installation]
Hi,
I am attempting to set up sunbird ed on AWS and am considering using the S3 service for storage. However, I am unable to locate any variables in the documentation related to S3 storage bucket. Can you please provide guidance on how to utilize an S3 bucket for storage?
Release Version: `release-7.0.0`

**Comment by gandham-santhosh** [comment]
@muzafferjoya Release-7.0 documentation is not published yet. You can refer for variables related to AWS S3 bucket https://github.com/project-sunbird/sunbird-devops/blob/release-7.0.0/private_repo/ansible/inventory/dev/Core/common.yml

**Reply by muzafferjoya** [reply]
Thanks @gandham-santhosh We will review and get back to you if necessary.

**Comment by HarishGangula** [comment]
As we haven't received a response from you, we will proceed to close this thread. If you have any further questions or require assistance in the future, please don't hesitate to start a new thread. Thank you.
